src/alembic/versions/2a6ea95342dc_blog_posts.py

```diff
@@ -0,0 +1,39 @@
+"""blog_posts
+
+Revision ID: 2a6ea95342dc
+Revises: 43f71e4bd6fc
+Create Date: 2024-08-31 03:06:11.516362
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '2a6ea95342dc'
+down_revision: Union[str, None] = '43f71e4bd6fc'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('blog_posts',
+    sa.Column('title', sa.String(length=128), nullable=False),
+    sa.Column('computing_id', sa.String(length=32), nullable=False),
+    sa.Column('date_created', sa.DateTime(), nullable=False),
+    sa.Column('last_edited', sa.DateTime(), nullable=False),
+    sa.Column('html_content', sa.Text(), nullable=False),
+    sa.Column('post_tags', sa.String(length=128), nullable=True),
+    sa.ForeignKeyConstraint(['computing_id'], ['officer_info.computing_id'], ),
+    sa.PrimaryKeyConstraint('title')
+    )
```

Comment:
```suggestion
    op.create_table('blog_posts',
        sa.Column('title', sa.String(length=128), primary_key=True, nullable=False),
        sa.Column('computing_id', sa.String(length=32), sa.ForeignKey("officer_info.computing_id"), nullable=False),
        sa.Column('date_created', sa.DateTime(), nullable=False),
        sa.Column('last_edited', sa.DateTime(), nullable=False),
        sa.Column('html_content', sa.Text(), nullable=False),
        sa.Column('post_tags', sa.String(length=128), nullable=True),
    )
```

see: https://github.com/CSSS/csss-site-backend/pull/78/files#diff-98a11339635fcc0df585c8fc64cd05ca369d9994dd56dd0317447475904854a6 for examples